src/vs/workbench/parts/debug/node/debugConfigurationManager.ts

```diff
@@ -152,7 +153,7 @@ const jsonRegistry = <jsonContributionRegistry.IJSONContributionRegistry>platfor
 jsonRegistry.registerSchema(schemaId, schema);
 
 export class ConfigurationManager implements debug.IConfigurationManager {
-
+	private settingsVariables: SettingsVariables;
```

Comment:
Just move it one line down, so we have nice seperation between public / private members